### PR TITLE
Announce route changes on navigation

### PR DESCRIPTION
**Pull request by microbit-robert**
See https://microbit-global.monday.com/boards/1356069004/views/10076885/pulses/1356379053

### CODE REVIEW

**Comment by microbit-robert** [comment]
This gives us translatable page titles and a route change announcement without 'migrating' to SvelteKit. The SvelteKit option seemed to be creating a new SvelteKit project and copying everything over with some config changes.

The title change is noticeable laggy, but the announcement is not. I'm not sure if this is the right implementation.

This doesn't work well for the "add data" page because we steal focus on an action input (maybe we shouldn't).

File: src/App.svelte [hunk 1]
```diff
@@ -51,6 +51,7 @@
 </script>
 
 <Router>
+  <div class="sr-only" id="announce-route" aria-live="polite" />
```
Comment (microbit-matt-hillsdon) [inline src/App.svelte:54]: Can we try putting the code that updates this in this file and making it more Svelte-y in terms of its store subscriptions?

Comment (microbit-robert) [reply src/App.svelte]: Moved and used `$:` syntax -> [04ac14c](https://github.com/microbit-foundation/ml-trainer/pull/165/commits/04ac14c919387ea9a69d84b4e3e46f38549ef150)

File: src/router/paths.ts [hunk 2]
```diff
@@ -26,3 +26,24 @@ export function navigate(path: PathType) {
   }
   currentPathPrivate.set(path);
 }
+
+const messageFormatter = get(format);
+const appName = 'micro:bit machine learning tool';
+export const getTitle = (path: PathType, t: typeof messageFormatter) => {
```
Comment (microbit-robert) [inline src/router/paths.ts:32]: Maybe this should just be `: any`. The i18n package doesn't export the MessageFormatter type (I don't think) so this is a workaround for that.

Comment (microbit-robert) [reply src/router/paths.ts]: Replicated the type we care about -> [f72eb2d](https://github.com/microbit-foundation/ml-trainer/pull/165/commits/f72eb2dad4edd3090078cde6496d68ce8c987bc9).

**Pull request merged**